project/Build.scala

```diff
@@ -775,14 +775,14 @@ object Build {
   lazy val dottyLibrarySettings = Seq(
     libraryDependencies += "org.scala-lang" % "scala-library" % scalacVersion,
     // Add version-specific source directories:
-    // - files in src-scala3 will only be compiled by dotty
-    // - files in src-scala2 will only be compiled by scalac
+    // - files in src-bootstrapped will only be compiled by dotty-bootstrapped
+    // - files in src-non-bootstrapped will only be compiled by dotty non-bootstrapped
```

Comment:
I would say "the reference compiler" (which currently is scala 2.12 and later will be another bootstrapped dotty)